airflow/providers/salesforce/hooks/salesforce.py

```diff
@@ -336,3 +348,44 @@ def object_to_df(
             df["time_fetched_from_salesforce"] = fetched_time
 
         return df
+
+    def write_schema_to_file(self, schema_file_name: str, obj: str, record_time_added: bool):
```

Comment:
are you certain schema cannot be inferred from query result? e.g. with pyodbc you can